src/settings/PatronNotices/tokens.js

```diff
@@ -89,6 +89,33 @@ const item = [
     allowedFor: [...Object.values(patronNoticeCategoryIds)],
   },
 ];
+const effectiveLocation = [
```

Comment:
`[...Object.values(patronNoticeCategoryIds)]` could you please move it to variable to avoid additional calculations?